.github/workflows/lint.yml

```diff
@@ -5,13 +5,12 @@ jobs:
   golangci-lint:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-go@v2
+      - uses: actions/setup-go@v3
         with:
-          go-version: "1.17"
+          go-version: 1.17
+      - uses: actions/checkout@v3
       - name: golangci-lint
-        uses: golangci/golangci-lint-action@v2
+        uses: golangci/golangci-lint-action@v3
         with:
-          skip-go-installation: true
-          version: v1.41.1
+          version: v1.29
```

Comment:
Are we downgrading the golangci-lint version? Whats the reason for doing so?

Reply:
So, the current github-action is not able to identify a few linters that golangci-lint support. I was testing out with different versions. Will fix this to the best compatible and highest version possible.

Reply:
It's fixed now.